modules/DRAFT-machineset-creating-azure-ultra-disk.adoc

```diff
@@ -0,0 +1,135 @@
+// Module included in the following assemblies:
```

Comment:
:x: IGNORE THIS FILE FOR NOW :x:
I will turn this into the PVC procedure after I have a chance to talk to the storage writer :)